bedrock/firefox/templates/firefox/browsers/mobile/focus.html

```diff
@@ -57,7 +57,7 @@
     media_include='firefox/browsers/mobile/includes/s2d-focus.html',
     media_after=True
   ) %}
-    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-focus">{{ ftl('mobile-focus-firefox-focus') }}</div>
+    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-focus{% if LANG=='de' %} mzp-t-product-klar{% endif %}">{{ ftl('mobile-focus-firefox-focus') }}</div>
```

Comment:
**To Do:** If this is waiting for https://github.com/mozilla/protocol/pull/932 before merging, please don't put both `mzp-t-product-focus` and `mzp-t-product-klar` on the element together. Focus could over-write klar if the includes are in the wrong order.
```suggestion
    <div class="mzp-c-wordmark mzp-t-wordmark-md {% if LANG=='de' %} mzp-t-product-klar{% else %}mzp-t-product-focus{% endif %}">{{ ftl('mobile-focus-firefox-focus') }}</div>
```

Reply:
ah good catch!

Reply:
@stephaniehobson Absolutely. That version was originally meant only as a hack until the proper Protocol change lands. (So it had to inherit most of the existing class, only overriding the url locally.) — Now with the wordmark component support it's either~or only. 👍 

@reemhamz Thank you so much for all the work upstream! It would definitely take me some time to figure it out properly (:) and I believe I'd have overcomplicated it nonetheless;))… While I can't add labels myself, you'd probably get a ping converting from draft anyways.